### PR TITLE
Feat/global footer

### PR DESCRIPTION
**Pull request by irfankhan10**
Add a global footer with fixed columns and column entries. Also added a link to Quantinuum Trademark guidelines.

image

### CODE REVIEW

File: src/custom/docs/components/footer/index.tsx [hunk 1]
```diff
@@ -1,12 +1,56 @@
 import { QuantinuumLogo } from "../../QuantinuumLogo";
 import { Separator } from "src";
-export const Footer = (props: { subtitle: string, columns: {
```
Comment (aidanCQ) [inline src/custom/docs/components/footer/index.tsx:3]: Subtitle here is used for product specific trademark lingo. 
Screenshot 2024-10-01 at 10 01 10

File: src/custom/docs/components/footer/text.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+[
```
Comment (aidanCQ) [inline src/custom/docs/components/footer/text.json:1]: What is this file used for?

Comment (irfankhan10) [reply src/custom/docs/components/footer/text.json]: deleted

**Comment by irfankhan10** [comment]
> Product specific links are still in use in other sites so we can't hardcode the entire footer.
> 
> Screenshot 2024-10-01 at 10 01 45

We agreed all global footers would contain non-customizable content.

File: src/custom/docs/components/footer/index.tsx (hunk 1 above)
Comment (irfankhan10) [reply src/custom/docs/components/footer/index.tsx:3]: Link added to the **Trademark Guidelines** instead

File: src/custom/docs/components/footer/index.tsx [hunk 3]
```diff
@@ -1,12 +1,56 @@
...
-}[] }) => {
+
+const footerConfig = {
+  columns: [
+    {
+      name: "Products",
+      items: [
+        {
+          name: "H-Series",
+          href: "https://docs.quantinuum.com/h-series"
```
Comment (aidanCQ) [inline src/custom/docs/components/footer/index.tsx:11]: We could probably remove the `https://docs.quantinuum.com` prefix for internal links.

File: src/custom/docs/components/footer/index.tsx [hunk 4]
```diff
@@ -1,12 +1,56 @@
...
+          name: "Careers",
+          href: "https://www.quantinuum.com/publications",
+        },{
+          name: "Events", 
+          href: "https://www.quantinuum.com/events"
+        }
+      ]
+    }
+  ],
+  subtitle: "",
```
Comment (aidanCQ) [inline src/custom/docs/components/footer/index.tsx:50]:
```suggestion
```

File: src/custom/docs/components/footer/index.tsx [hunk 5]
```diff
@@ -23,14 +67,26 @@ export const Footer = (props: { subtitle: string, columns: {
...
+            Cookie Notice
+          </a><div>/</div>
+          <a target="_blank" href="https://cdn.prod.website-files.com/669960f53cd73aedb80c8eea/66d87306d9caeeb12196f409_666c39372d2febd0b0bedf71_Quantinuum%20Trademark%20Guidelines%20v1.1%20June%202024.pdf" className='font-medium text-xs tracking-tight text-blue-600 dark:text-blue-300'>
+            Trademark Guidelines
+          </a>
         </div>
       </div>
       <div className="grid grid-cols-1 gap-16 md:grid-cols-3 md:gap-24">
-        {props.columns.map((col) => {
+        {footerConfig.map((col) => {
```
Comment (aidanCQ) [inline src/custom/docs/components/footer/index.tsx:89]:
```suggestion
        {footerConfig.columns.map((col) => {
```

File: src/custom/docs/components/footer/index.tsx [hunk 6]
```diff
@@ -23,14 +67,26 @@ export const Footer = (props: { subtitle: string, columns: {
...
           return (
             <div key={col.title} className="flex flex-col md:items-end">
```
Comment (aidanCQ) [inline src/custom/docs/components/footer/index.tsx:91]:
```suggestion
            <div key={col.name} className="flex flex-col md:items-end">
```

**Review by aidanCQ — APPROVED** [review]

**Pull request merged**